src/screens/VisualTests/NoBuild.tsx

```diff
@@ -53,15 +53,16 @@ export const NoBuild = ({
   hasData,
   hasProject,
   hasSelectedBuild,
-  startDevBuild,
   localBuildProgress,
   branch,
 }: NoBuildProps) => {
   const { setAccessToken } = useAuthState();
+  const { isRunning, startBuild, stopBuild } = useRunBuildState();
+
   const getDetails = () => {
     const button = (
-      <Button size="medium" variant="solid" onClick={startDevBuild}>
-        <PlayIcon />
+      <Button size="medium" variant="solid" onClick={isRunning ? stopBuild : startBuild}>
+        {isRunning ? <StopAltIcon /> : <PlayIcon />}
```

Comment:
Do we have any stories that cover this?

Reply:
Turns out this state isn't actually reachable, as the button won't render at all while the build is running.